test/bom.bats

```diff
@@ -0,0 +1,125 @@
+load helpers
+
+function setup() {
+    stacker_setup
+}
+
+function teardown() {
+    cleanup
+}
+
+@test "all container contents must be accounted for" {
+  cat > stacker.yaml <<EOF
+bom-parent:
+    from:
+        type: oci
+        url: $CENTOS_OCI
+    bom:
+      generate: true
+      packages:
+      - name: pkg1
+        version: 1.0.0
+        license: Apache-2.0
+        paths: [/pkg1]
+      - name: pkg2
+        version: 1.0.0
+        license: Apache-2.0
+        paths: [/pkg2]
+    run: |
+      # discover installed pkgs
+      /stacker/tools/static-stacker --internal-userns internal-go bom-discover
```

Comment:
The mechanism that you had before for executing stacker-bom was much cleaner.  we should not expect a user to use '--internal-userns internal-go' flags (we should never document 'internal-go' flags).

We want the documented command that the user runs to be one of:

 1. stacker bom-discover
 2. stacker-bom discover
 3. stacker bom discover

I think i prefer 3, and making 'stacker bom discover' just work wherever it was run.

I think the right way to fix that is to add the bom commands to main (not internal-go), and then make shouldSkipInternalUserns (cmd/stacker/main.go) return false for 'bom' commands.  stacker bom subcommand never needs to enter a internal user namespace. It just inspects directory trees and writes content somewhere.